src/generators/generate-thunkify.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2016 Shape Security, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const spec = require('shift-spec').default;
+const { sanitize, parameterize, isStatefulType } = require('../utilities.js');
+
+function force({ name, type }) {
+  switch (type.typeName) {
+    case 'Maybe':
+      return `${sanitize(name)} === null ? null : ${sanitize(name)}()`;
```

Comment:
Do we actually distinguish `null` and `undefined` in our AST? Either way, we should probably make this a bit more permissive and use `==`.

Reply:
We [do](https://www.w3.org/TR/WebIDL/#idl-nullable-type). But I'm fine relaxing it here, I suppose.